app/components/pages/SubmissionWizard/steps/Files/SupportingUpload.js

```diff
@@ -152,7 +152,9 @@ class SupportingUpload extends React.Component {
       files: props.files.map((file, index) => ({
         file,
         id: index,
-        success: true,
+        success: file.status === 'STORED',
```

Comment:
Cory and I are making a note - that when we eventually write to the ApolloCache we should cosolidate this into a clientStatus (i.e. calculated from the server status)